python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -757,14 +764,28 @@ def runner_data(self) -> Dict:
 
     def _get_trial_paths(self) -> List[str]:
         if self.trials:
-            _trial_paths = [t.logdir for t in self.trials]
+            # Get the relative paths from the trials to allow
+            # for changes in the self._local_base_dir.
+            _trial_paths = [
+                str(self._local_base_dir.joinpath(t.rel_logdir))
+                if t.rel_logdir
+                else t.rel_logdir
```

Comment:
Should this be
```suggestion
                else t.logdir
```
?

Reply:
Maybe just `hasattr(t, "rel_logdir")` for the if clause?

Reply:
I have used your suggestions, @krfricke